src/components/Header/Header.js

```diff
@@ -125,26 +122,19 @@ class Header extends React.Component {
   }
 
   componentWillUnmount() {
-    HeaderStore.unlisten(this.onChange.bind(this));
     // Listen on FeatureFlags Store updates
     FeatureFlags.store.unlisten(this.onFeatureFlagsChange.bind(this));
     // Removing event listener to minimize garbage collection
   }
 
   onChange() {
-    this.setState(
-      _extend(
-        {
-          loginCookieValue: this.state.loginCookieValue,
-          patronName: this.state.patronName,
-          patronInitial: this.state.patronInitial,
-          patronDataReceived: this.state.patronDataReceived,
-          isFeatureFlagsActivated: {},
-          logOutUrl: this.state.logOutUrl,
-        },
-        HeaderStore.getState()
-      )
-    );
+    this.setState({
```

Comment:
do you even need an onChange method? we aren't expecting anything to trigger it. The login state objects might but setState() will take care of that right?

Reply:
You're right. Removing `onChange`. Thanks @nonword  and @rhernand3z